lib/helpers/data_parsers.rb

```diff
@@ -15,4 +15,8 @@ def verify_signature(payload_body)
     signature = 'sha1=' + OpenSSL::HMAC.hexdigest(OpenSSL::Digest.new('sha1'), settings.github_secret, payload_body)
     throw(:halt, [500, "Signatures didn't match!\n"]) unless Rack::Utils.secure_compare(signature, request.env['HTTP_X_HUB_SIGNATURE'])
   end
+
+  def payload
```

Comment:
I'm not sure about this.  It's only very slightly more convenient than using env['parsed_body'] and there might be a better way.  I'm going to see what https://github.com/aars/rack-bodyparser#patch-rackrequest is all about.

Reply:
I like this. Even if it's not that much shorter, it's much clearer naming.

Reply:
Cool.  I did look at https://github.com/aars/rack-bodyparser#patch-rackrequest, but it didn't really seem much of an improvement.  Instead of `env['parsed_body']` you could use eg `request.payload`.